doc/whats_new/v0.19.rst

```diff
@@ -1066,4 +1066,3 @@ Dai, Greg Stupp, Grzegorz Szpak, Bertrand Thirion, Hadrien Bertrand, Harizo
 Rajaona, zxcvbnius, Henry Lin, Holger Peters, Icyblade Dai, Igor
 Andriushchenko, Ilya, Isaac Laughlin, Iván Vallés, Aurélien Bellet, JPFrancoia,
 Jacob Schreiber, Asish Mahapatra
-
```

Comment:
Can we leave this newline here? Most files end with a newline and it's consistent with the Python files.

Reply:
I think there is still a newline there. The pre-commit formatter complained about the file having 2 newlines at the end, and removed 1 of them.

Reply:
Ah, I see it when I pull the PR locally. Strange how the GitHub interface makes it look like there is no more newline.